insights/parsers/mount.py

```diff
@@ -118,3 +120,25 @@ def parse_content(self, content):
             self.rows.append(entry)
             if match:
                 self.mounts[mount['mount_point']] = entry
+
+    def search(self, **kwargs):
+        """
+        Returns a list of the mounts (in order) matching the given criteria.
+        Keys are searched for directly - see the ``keyword_search`` utility
```

Comment:
@PaulWay it would be helpful to add a reference to this function instead of just highlighting the name, for instance:
```
:py:func:`insights.parsers.keyword_search`
```

Reply:
OK, done!